.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+    "connector": {
+      "name": "local",
```

Comment:
- [ ] It would be better if we use a deployed link instead of the local host we have currently and also instead of removing the MIT Iicense, I will recommend you fix the link so that our repo looks professional